data/persistence/build.gradle.kts

```diff
@@ -12,6 +12,10 @@ sqldelight {
         packageName = "social.androiddev.common.persistence"
         sourceFolders = listOf("sqldelight")
     }
+    database("TimelineDatabase") {
```

Comment:
shouldn't this be collocated with a timeline feature, i.e `ui/timeline/src(...)/data` ?

Reply:
I think there should be feature/data and feature/ui for most features, let me do as a follow up if everyone agrees